app/assets/stylesheets/application.scss

```diff
@@ -13,7 +13,10 @@
  */
 @import "bootstrap";
 
-@import "scaffolds";
+@import "general";
+@import "containers";
+
 @import "books";
```

Comment:
suggestion: Should remove `books` import if there is no styling to include. Same with `home`